pkg/utils/filter_map.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Can you add the "chore:" prefix to commit "Add generic FilterMap util function"?